src/public/peoplePicker.ts

```diff
@@ -0,0 +1,97 @@
+import { ensureInitialized, isAPISupportedByPlatform } from '../internal/internalAPIs';
+import { FrameContexts } from './constants';
+import { ErrorCode, SdkError } from './interfaces';
+import { validatePeoplePickerInput } from '../internal/mediaUtil';
+import { sendMessageToParent } from '../internal/communication';
+
+export namespace peoplePicker {
```

Comment:
Can we just use "people" as the namespace here? The namespace should really be more generic so it can encompass a wider variety of related APIs.

Reply:
done